infer_video.py

```diff
@@ -199,16 +199,28 @@ def paint_thread_func(show=True, keep_video_at=""):
     exit()
 
 
-def score_thread_func():
+def score_thread_func(groundtruth, allclasses):
     scores = []
     # cv2.namedWindow("scorer debug")
     while True:
         orig_image = orig_image_queue.get()
         if orig_image == "kill":
             break
-        orig_label_np = orig_image['label']
-        if len(orig_label_np.shape) == 3:
-            orig_label_np = orig_label_np[:, :, 0]
+        if groundtruth=='label':
```

Comment:
ideally we should just be generating these labels beforehand, otherewise this is a lot of overhead. leaving this in for now though